azure-pipelines.yml

```diff
@@ -63,12 +63,16 @@ stages:
       jobGroups:
       - name: broadcom
         variables:
+          dbg_image: yes
```

Comment:
How about using the variable INSTALL_DEBUG_TOOLS=y, do we want to build release version and the debug version for PR build, or the debug version is good enough for PR build?